server/container/guice/cassandra-guice/src/main/java/org/apache/james/jmap/CassandraMethodsModule.java

```diff
@@ -41,7 +41,7 @@
 import com.google.inject.multibindings.Multibinder;
 import com.google.inject.name.Names;
 
-public class MethodsModule extends AbstractModule {
```

Comment:
There's an easy fix for that : declare MethodsModule like that `MethodsModule<Id extends MailboxId>`
and replace CassandraId below with Id.